base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -451,3 +457,36 @@ func (vgm *virtualGroupManager) releaseSPAndGVGLoop() {
 		vgm.freezeSPPool.ReleaseSP()
 	}
 }
+
+func isSPEndpointValid(endpoint string) bool {
```

Comment:
1. Background detection of sp survival
2. The front-end process is only a query, and it will be actively queried when it does not exist; reducing the possibility of blocking the front-end process
Maybe better.